code/game/machinery/droneDispenser.dm

```diff
@@ -22,7 +22,7 @@
 
 	var/droneMadeRecently = 0
 	var/cooldownTime = 1800 //3 minutes
-	var/dispense_type = /obj/item/drone_shell //The item the dispenser will create
```

Comment:
Was this intentional? Just making sure, I assume safe drones were the ones players were able to find.